packages/components/src/components/autocomplete/autocomplete.tsx

```diff
@@ -105,8 +106,8 @@ const Autocomplete = React.memo((props: TAutocompleteProps) => {
             let new_filtered_items = [];
 
             if (is_list_visible) {
-                if (typeof other_props.onSearch === 'function') {
-                    new_filtered_items = other_props.onSearch(value.toLowerCase(), list_items);
+                if (typeof onSearch === 'function') {
```

Comment:
what's the specification of writing like this? 
And may I know which console error is being resolved?

Reply:
i just destructured it. probably it was there for some reason since it is a shared component.

Reply:
Seems like it is. yet I'm still confused what was the console error being produced due to this 😄

Reply:
ah.. about the console error.. it was a props validation error.. when we were not destructuring it, we were completely passing all the {..other_props} to the input component used inside the autocomplete component. but since then the actual <input> didn't have the onSearch attribute, it threw a warning for 'onSearch' property does not exist on input .. so i destructured only the onSearch out of it.

Reply:
oh cool. Thanks 💯

Reply:
I would suggest to add this error message to the description so it'll be easier to review the scope 🎉